src/main/java/mate.academy/service/impl/ProductServiceImpl.java

```diff
@@ -2,13 +2,18 @@
 
 import java.util.List;
 import java.util.stream.Collectors;
+import mate.academy.lib.Component;
+import mate.academy.lib.Inject;
 import mate.academy.model.Product;
 import mate.academy.service.FileReaderService;
 import mate.academy.service.ProductParser;
 import mate.academy.service.ProductService;
 
+@Component
 public class ProductServiceImpl implements ProductService {
+    @Inject
     private ProductParser productParser;
```

Comment:
Consider giving a more informative name to the 'productParser' variable, indicating what kind of parser it is or what format it parses, to improve code readability.

Reply:
--